src/test/spec/unified_runner/operation.rs

```diff
@@ -1644,6 +1649,147 @@ impl TestOperation for RunCommand {
     }
 }
 
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+pub(super) struct RunCursorCommand {
+    command: Document,
+    // We don't need to use this field, but it needs to be included during deserialization so that
+    // we can use the deny_unknown_fields tag.
+    #[serde(rename = "commandName")]
+    _command_name: String,
+    selection_criteria: Option<SelectionCriteria>,
+    cursor_type: Option<String>,
+    batch_size: Option<u32>,
+    max_time_m_s: Option<u64>,
+    comment: Option<Bson>,
```

Comment:
If you take a look at most of our other operation implementations you'll see we use `#[serde(flatten)]` to deserialize options directly into their respective options structs rather than specifying them at the top-level. This makes it easier to introduce new options in the future. Can we update this implementation to do that instead?